bundles/core/src/test/resources/searchlister/searchlister-v1-withResults.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "searchlister-fb0b07e803",
+    "title": "Sample Title",
+    "layout": "Custom",
+    "elements": {
+        "pagination": {
+            "loadAction": "/?searchText=Search+Text&limit=10&description=Desc&orderby=title&sort=asc&title=Sample+Title&tags=full%2Fpath%2Fto%2Ftag&offset=1",
```

Comment:
Why is this called as loadAction ? What does this mean

Reply:
This is for loading next page of results 
(load more button in the UI)